chroma-manager/tests/framework/utils/cluster_setup.sh

```diff
@@ -38,7 +38,7 @@ for key in CentOS-7 redhat-release; do
     fi
 done
 
-if [[ \$HOSTNAME = *vm*2 ]]; then
+if [[ \$HOSTNAME = *vm*2 ]] || [[ \$HOSTNAME = $CLIENT_1 ]]; then
```

Comment:
How/where is `$CLIENT_1` being set?

Reply:
in
chroma-manager/tests/utils/json_cfg2sh.py:    print "CLIENT_1=\"%s\"" % config["lustre_clients"][0]['address']

Reply:
Note however, in: https://github.com/intel-hpdd/intel-manager-for-lustre/blob/14be42727b015aed13b9c647828d20850e656e37/chroma-manager/tests/utils/json_cfg2sh.py#L38-L39
`$CLIENT_1` is not always defined.  This change should account for that and should actually complete the fix by getting rid of the `[[ \$HOSTNAME = *vm*2 ]]` hack completely.